jobs/JGLOBAL_SNOW_ENSEMBLE_ANALYSIS

```diff
@@ -0,0 +1,60 @@
+#! /usr/bin/env bash
+
+source "${HOMEgfs}/ush/preamble.sh"
+source "${HOMEgfs}/ush/jjob_header.sh" -e "esnowanl" -c "base esnowanl"
+
+##############################################
+# Set variables used in the script
+##############################################
+# Ignore possible spelling error (nothing is misspelled)
+# shellcheck disable=SC2153
+GDUMP="gdas"
+GDUMP_ENS="enkfgdas"
+export GDUMP
+
+##############################################
+# Begin JOB SPECIFIC work
+##############################################
+# Generate COM variables from templates
+YMD=${PDY} HH=${cyc} declare_from_tmpl -rx \
+    COMIN_OBS:COM_OBS_TMPL \
+    COMOUT_ATMOS_ANALYSIS:COM_ATMOS_ANALYSIS_TMPL \
+    COMOUT_CONF:COM_CONF_TMPL
+MEMDIR="ensstat" YMD=${PDY} HH=${cyc} declare_from_tmpl \
+    COMOUT_SNOW_ANALYSIS:COM_SNOW_ANALYSIS_TMPL
+
+mkdir -p "${COMOUT_SNOW_ANALYSIS}" "${COMOUT_CONF}"
+
+for imem in $(seq 1 "${NMEM_ENS}"); do
+    memchar="mem$(printf %03i "${imem}")"
+    MEMDIR=${memchar} RUN=${GDUMP_ENS} YMD=${PDY} HH=${cyc} declare_from_tmpl \
+        COMOUT_SNOW_ANALYSIS:COM_SNOW_ANALYSIS_TMPL
```

Comment:
`GDUMP_ENS=gdas`.
When this script is run with `RUN=enkfgfs`, this will create a variable in the `enkfgdas` path for `COMOUT`.
Is that correct?
Do we plan to execute `JGLOBAL_SNOW_ENSEMBLE_ANALYSIS` in the early cycle with `RUN=enkfgfs`?

Reply:
I think you and I chatted about this previously, as we don't run an early cycle enkfgfs fcst, there's no need to do this, the GEFS can decide if they wish to do this recentering?

Reply:
In that case, we need to name this job `JENKFGDAS_` instead of `JGLOBAL_` to conform with `JRUN_` convention.

Reply:
is it `JGDAS_ENKF` to be consistent with the others or `JENKFGDAS_` ?